src/App.tsx

```diff
@@ -114,19 +96,4 @@ function Break() {
   return <div style={{ width: "50px" }} />;
 }
 
-function Title() {
-  return (
-    <h1
-      style={{
-        textAlign: "center",
-        marginBottom: "50px",
-        marginTop: "20px",
-        fontFamily: "sans-serif",
```

Comment:
Please do not give font family to elements separately, unless there is a very special case and reason. There must be a one source of truth and you should give font family globally. Please see the updated `App.css` file.

Reply:
make sense, thank you